tests/test_decoders.py

```diff
@@ -137,7 +137,7 @@ def test_decoding_errors(header_value):
         for chunk in data:
             yield chunk
 
-    response = httpx.Response(200, content=iterator())
+    response = httpx.Response(200, stream=iterator())
```

Comment:
Strictly speaking these would need to be `stream=AsyncIteratorStream(iterator())`, correct?

I'd be in favor of making that change, since we're otherwise setting false expectations that `stream=` accepts async generators when it actually doesn't (it just so happens to work because responses don't call into `.get_headers()` or `.can_replay()`).

Reply:
100% yes! I’d missed these - thanks for the catch!